api/tests/routes/backoffice/pro_test.py

```diff
@@ -1098,25 +1076,27 @@ def test_connect_as_user_invalid_object_type(self, authenticated_client, legit_u
             == "Échec de la validation de sécurité, veuillez réessayer"
         )
 
-    @override_features(WIP_CONNECT_AS=True)
     def test_connect_as_user_not_found(self, authenticated_client):
         form_data = {"object_type": "user", "object_id": 0, "redirect": "/"}
         response = self.post_to_endpoint(
             authenticated_client,
             form=form_data,
-            expected_num_queries=self.expected_num_queries + 1,  # +1 for rollback query
+            # +1 for rollback query
+            # -1 for Feature flags
+            expected_num_queries=self.expected_num_queries,
```

Comment:
Par rapport à la précédente remarque, je présume qu'on devrait donc ainsi avoir ici, et dans les méthodes de test suivantes : 
```suggestion
            # +1 for rollback query
            expected_num_queries=self.expected_num_queries + 1,
```